boards/arm/thingy53_nrf5340/Kconfig.defconfig

```diff
@@ -8,6 +8,9 @@ if BOARD_THINGY53_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP_NS
 config BOARD
 	default "thingy53_nrf5340_cpuapp"
 
+config BOOTLOADER_MCUBOOT
+	default y if !MCUBOOT
```

Comment:
Wouldn't just `default y` be sufficient when the `BOOTLOADER_MCUBOOT` option itself: https://github.com/nrfconnect/sdk-zephyr/blob/c6184a9ff7382e5c699c35b11c9b4e096a7f247d/Kconfig.zephyr#L749?

Reply:
no, cause dependencies are not inherited.

So this needs either the current `if !MCUBOOT` or a `depends on !MCUBOOT`

Reply:
Removing this dependency results in adding MCUboot child image in a never ending-loop (MCUboot child image seems to add MCUboot child image recursively).